salesforce-composite/2.6/modules/ROOT/pages/salesforce-composite-connector-reference.adoc

```diff
@@ -0,0 +1,1199 @@
+= Salesforce Composite Connector 2.6 Reference - Mule 4
+:page-aliases: connectors::salesforce/salesforce-composite-connector-reference.adoc
+
+Support Category: Select
```

Comment:
Let's add the link:
Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]